client/post-editor/editor-revisions-list/style.scss

```diff
@@ -1,6 +1,3 @@
-// @TODO we should decouple from accordion styles
-@import 'components/accordion/style';
```

Comment:
this is probably the scariest part.. does the revision list rely on any of the accordion styles by name?

Reply:
nope. they were just using the one bit.